biogtr/io/association_matrix.py

```diff
@@ -83,16 +93,18 @@ def numpy(self) -> np.ndarray:
         return self.matrix
 
     def to_dataframe(
-        self, row_label: str = "gt", col_label: str = "gt"
+        self, row_labels: str = "gt", col_labels: str = "gt"
     ) -> pd.DataFrame:
         """Convert the association matrix to a pandas DataFrame.
 
         Args:
-            row_label: How to label the rows(queries).
-                        If `gt` then label by gt track id.
-                        If `pred` then label by pred track id.
+            row_labels: How to label the rows(queries).
+                        If list, then must match # of rows/queries
+                        If `"gt"` then label by gt track id.
+                        If `"pred"` then label by pred track id.
```

Comment:
Just needs to be one indentation level (4 spaces) to the right from the starting column of `row_labels`.